packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -681,3 +682,22 @@ function getDotDevDotVarsContent(
 
 	return null;
 }
+
+/**
+ * Returns true if the `changedFile` matches one of a potential .dev.vars file.
+ */
+function hasDotDevDotVarsFileChanged(
+	resolvedPluginConfig: ResolvedPluginConfig,
+	changedFile: string
+) {
+	const dotDevDotVarsPaths = new Set(
+		[...resolvedPluginConfig.configPaths].map((configPath) =>
+			path.join(path.dirname(configPath), ".dev.vars")
```

Comment:
Do we need to handle `.dev.vars.<env>`  type files too?

Reply:
oh.... that's a good question! 🤔  I need to check, I wonder if we support them in the first place

Reply:
yes, we do! great call, I'll address this, thanks! 🫶

Reply:
done 🙂